### ISSUE TITLE
Users should be able to navigate around the app using keyboard

### ISSUE THREAD

**Issue by mehmetf** [flutter/flutter#13264]
None of the navigational shortcuts work. I should be able to navigate Flutter apps installed on ChromeOS using arrows/tab/enter key.s The only key that works is Esc which quits the app.

**Comment by mehmetf** [comment]
A related issue: On forms, pressing tab should switch to the next field.

**Comment by eseidelGoogle** [comment]
@goderbauer do you think this is related to other AX work or separate?

**Comment by goderbauer** [comment]
This is somewhat related to AX, but has not been on my radar so far.

**Comment by mehmetf** [comment]
@eseidel Please set appropriate milestone for this so we can deliver it by August 15, 2018. This is a hard deadline. If we don't meet this, Flutter apps published in Google will be banned on tablets (not just Chromebooks).

**Comment by jslavitz** [comment]
How exactly are those buttons supposed to interact with the apps? Arrows should scroll I assume, but what should the other keys map to?

**Comment by mehmetf** [comment]
@jslavitz Here are some public resources that might help:

https://developer.android.com/topic/arc/optimizing

This talks about facets of supporting Chromebook within Android apps. There are links off of this to specific topics such as [tab navigation](https://developer.android.com/training/keyboard-input/navigation#Tab).

**Comment by jslavitz** [comment]
Also, is this the same issue as #15756?

**Comment by jeowen** [comment]
Writing on behalf of developers working for the US Department of Veterans Affairs.  We are hoping to develop federal applications that serve hundreds of thousands of U.S. Veterans and service members, but all of our applications require adherence to strict accessibility standards. Hoping this issue is something that can be addressed so that we, and the many Veterans who require hardware to utilize mobile applications, can use Flutter.  We recently developed a pilot app with Flutter & the app was rejected by 508 review because of this issue.

**Comment by csells** [comment]
@jeowen I'm the PM for the Flutter developer experience. I'd love to talk to you about how Flutter can help you in general. Feel free to reach out to csells@google.com.

**Comment by jeowen** [comment]
Thanks @csells just sent you an email & looks like there has been an update.  Do you know if this was addressed?

**Comment by hkwinslow** [comment]
@jeowen Was this ever addressed for you?

**Comment by gmoraja** [comment]
I'll answer for @jeowen. Yes, all the blocking issues in our apps have been addressed. Thank you.
